### ISSUE TITLE
i18n dependancy incompatible with rails 5.2-rc1

### ISSUE THREAD

**Issue by renchap** [svenfuchs/rails-i18n#766]
In https://github.com/rails/rails/pull/31991 Rails dependency on `i18n` has been bumped to `~> 1.0`.
This makes `rails-i18n` incompatible as it depends on `~> 0.7`:

```
Bundler could not find compatible versions for gem "i18n":
  In Gemfile:
    activesupport was resolved to 5.2.0.rc1, which depends on
      i18n (~> 1.0)

    rails-i18n (~> 5.0.0) was resolved to 5.0.4, which depends on
      i18n (~> 0.7)
```

The Rails change has been backported to all supported versions, so it will cause problems with this gem as soon as they are released.

The change should be simple: bump the dep to 1.0. From what @radar said in the PR above, he released 1.0 because it breaks compat with old rubies and this is the only APi change, so I guess the fix is simply to change the dependancy in this gem's `gemspec`.

**Comment by kuroda** [comment]
I just released `rails-i18n` 5.1.0 bumping the `i18n` version range:

```
  s.add_runtime_dependency('i18n', '>= 0.7', '< 2')
```

**Comment by renchap** [comment]
Thanks, I can confirm it solves the issue!
